routes/users.js

```diff
@@ -175,80 +178,196 @@ router
 router
   .route("/profile")
   //TODO: choose how to display user details
-  .put(async (req, res) => {
+  .get(async (req, res) => {
```

Comment:
revert to  post to put wherever it was changed we can add the middleware for your code if needed